quickstart/202-machine-learning-moderately-secure-existing-VNet/compute.tf

```diff
@@ -12,7 +12,7 @@ resource "azurerm_machine_learning_compute_instance" "compute_instance" {
   location                      = azurerm_resource_group.default.location
   machine_learning_workspace_id = azurerm_machine_learning_workspace.default.id
   virtual_machine_size          = "STANDARD_DS2_V2"
-  subnet_resource_id            = var.training_subnet_resource_id
+  subnet_resource_id            = "${data.azurerm_subnet.training.id}"
```

Comment:
This does not need to be in the string format. it can just be
```terraform
subnet_resource_id = data.azurerm_subnet.training.id
```

This applies to `subnet_resource_id` as well as other `data.azurerm_*` references below.

Reply:
looks like this was resolved in a later merge so no longer applicable.